content/lxc/downloads.md

```diff
@@ -14,16 +14,16 @@ For Ubuntu users, we have official PPAs for LXC:
 And for those who want development snapshots:
 
  * [lxc-git-master](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-master): "master" branch
- * [lxc-git-stable-3.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-3.0): "stable-3.0" branch
  * [lxc-git-stable-4.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-4.0): "stable-4.0" branch
+ * [lxc-git-stable-5.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-5.0): "stable-5.0" branch
```

Comment:
@ru-fu @stgraber link doesn't exist.

Reply:
Should it exist or should we remove the link?

Reply:
What do you think @stgraber ?

Reply:
It exists now (empty like the 4.0 one though as I don't have time to sort out those builds until we fully move to Github Actions).

Reply:
thanks!